scripts/buildArangoDB6.fish

```diff
@@ -66,7 +66,7 @@ end
 if test "$SAN" = "On"
   # Suppress leaks detection only during building
   set -gx SAN_OPTIONS "detect_leaks=0"
-  set -l SANITIZERS "-fsanitize=address -fsanitize=undefined -fsanitize=float-divide-by-zero -fsanitize=leak"
+  set -l SANITIZERS "-fsanitize=address -fsanitize=undefined -fsanitize=float-divide-by-zero -fsanitize=leak -fno-sanitize-address-use-after-scope"
```

Comment:
detect_stack_use_after_return=false
We disable it in runtime.
Better to disable in compile time
https://github.com/arangodb/oskar/blob/master/scripts/lib/tests.fish#L32